app/components/RunDisplay.vue

```diff
@@ -25,12 +27,27 @@ defineProps<{
       {{ error }}
     </div>
 
+    <div
+      v-if="runs.length > 0"
+      class="controls-container"
+    >
+      <label class="toggle-raw-headers-label">
```

Comment:
it's still really ugly:

<img width="296" height="60" alt="Image" src="https://github.com/user-attachments/assets/0c09ac4a-498c-4fe5-a25b-b9f4bf3d3188" />

maybe you aren't taking into account the global styles the app loads in the nuxt config file? maybe your permissions are blocking it from loading or something?

now that I think about it, maybe the "Reset" button and this toggle should be grouped together somehow? they're both controls. maybe make a visually nice controls section somewhere? the only constraint is it shouldn't take much vertical space, ideally zero somehow.